assets/lokomotive-kubernetes/bare-metal/flatcar-linux/kubernetes/cl/controller.yaml.tmpl

```diff
@@ -171,6 +173,19 @@ storage:
             --net=host \
             --dns=host \
             --exec=/bootkube -- start --asset-dir=/assets "$@"
+    - path: /etc/kubernetes/configure-kubelet-cgroup-driver
```

Comment:
You can mention in the commit message that we do this for all the other platforms.

Reply:
Improved the commit message. PTAL.